email_mailbox/src/components/PanelWrapper.js

```diff
@@ -398,8 +409,8 @@ class PanelWrapper extends Component {
         this.props.onUpdateUnreadEmailsBadge(labelIdsBadge);
     }
 
-    if (activity || label) {
-      this.props.onAddDataApp({ activity, label });
+    if (activity || label || deletedLabels) {
+      this.props.onAddDataApp({ activity, label, deletedLabels });
```

Comment:
wrong: this cross... label and deleteLabels are from the same reducer, jusy one action should happen.

Reply:
Should I add something like an ADD_DELETE_LABELS action instead?